aws/lambda/node/README.md

```diff
@@ -0,0 +1,21 @@
+# Node Lambdas
+
+## Running Locally
+
+### Setup
+Install [aws-sam-cli](https://github.com/awslabs/aws-sam-cli/blob/master/docs/installation.rst)
```

Comment:
this documentation makes it seem like you have to mess around with paths, but if you pip install it into a virtualenv it just works. So we could add it to our requirements.txt in scripts.